mmdeploy/apis/extract_model.py

```diff
@@ -63,140 +62,6 @@ def extract_model(model: Union[str, onnx.ModelProto],
     Returns:
         onnx.ModelProto: The extracted sub-model.
```

Comment:
I think that you want to remove the terminology `sub-model`, this one is missed.

Reply:
Done